build-system/babel-config/pre-closure-config.js

```diff
@@ -83,7 +83,7 @@ function getPreClosureConfig() {
     !isCheckTypes
       ? './build-system/babel-plugins/babel-plugin-transform-json-configuration'
       : null,
-    argv.esm
+    !(argv.fortesting || isCheckTypes)
```

Comment:
Why this change?

Reply:
This was an intentional change that I remember making. Without it, it was impossible to get `v0.js` to be byte-for-byte identical before and after #27683. See https://github.com/ampproject/amphtml/pull/27683#issuecomment-611822634

Having said that, I'm going to defer to your knowledge of `babel` transforms and when they should be run. Can you confirm that this doesn't affect the minified output in undesirable ways?